src/applications/static-pages/renderHomepageBanner.js

```diff
@@ -0,0 +1,51 @@
+import yaml from 'js-yaml';
+import * as Sentry from '@sentry/browser';
+
+const HOMEPAGE_BANNER_YML_LOCATION =
+  'https://raw.githubusercontent.com/department-of-veterans-affairs/vagov-content/master/fragments/home/banner.yml';
+
+async function loadConfigFromGitHub() {
+  const response = await fetch(HOMEPAGE_BANNER_YML_LOCATION, {
+    mode: 'cors',
+  });
+
+  const homepageBannerYml = await response.text();
+  return yaml.safeLoad(homepageBannerYml);
+}
+
+function renderToDocument(banner) {
+  if (!banner.visible) {
+    return;
+  }
+
+  const root = document.getElementById('homepage-banner');
+
+  root.innerHTML = `
```

Comment:
ooooh, danger zone

I know you're injecting HTML from the YAML, but... do we have a template system that'll sanitize this at all? i.e. angular's template system allows injecting HTML but still strips out potential JS

Reply:
Yeah...I suppose I should implement something like [sanitize-html](https://www.npmjs.com/package/sanitize-html). I held off because of the extra dependency weighed against knowing that access to this content is managed by GitHub. But I suppose someone could gain access to a couple GitHub accounts that has access to the content repo. I'll patch this.